dom.js

```diff
@@ -36,6 +37,9 @@ module.exports = function (/*options*/) {
 	return function (deck) {
 		var update, current, resolvedQuery;
 
+		/* If `classes` hasn't been initialized jet, do it now */
+		if (!deck.parent.classList.contains('parent')) classes()(deck);
```

Comment:
Of course it should. Sorry :)

I just read `addClass(deck.parent, 'parent')` in the source code of `bespoke-classes` and didn't properly read what `addClass` actually did.

Fixing it...